website/src/components/ConformanceShowcase/styles.module.css

```diff
@@ -0,0 +1,27 @@
+.conformanceShowcase {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin: auto;
+}
+
+.conformanceShowcaseItem {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    text-align: left;
+}
+
+.conformanceImage img {
+    max-width: 25rem;
+    min-width: 25rem;
+    max-height: 8rem;
+}
+
```

Comment:
Should probably only have 1 empty line for consistency. This is the kind of thing prettier or stylelint would probably clean up automatically.